Aptoide/aptoide-client-v8
# [APP-2504] - Update Support Screen UI

**What does this PR do?**

It updates the Support Screen UI according to Aptoide Games design.  

P.S. I've also changed the email to the aptoide.games zendesk and the placeholder string to say "Aptoide Games" instead of "Games Hub"

**Database changed?**

No

**Where should the reviewer start?**

- [ ] SupportScreen.kt
- [ ] SupportActivity.kt
- [ ] strings.xml

**How should this be manually tested?**

  Test the support screen from settings and see if everything matches the design.

  Flow on how to test this or QA Tickets related to this use-case: [APP-2504](https://aptoide.atlassian.net/browse/APP-2504)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-2504](https://aptoide.atlassian.net/browse/APP-2504)



**Code Review Checklist**


[APP-2504]: https://aptoide.atlassian.net/browse/APP-2504?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APP-2504]: https://aptoide.atlassian.net/browse/APP-2504?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
